main.py

```diff
@@ -278,60 +278,62 @@ def ui_access_wrapper(self, *args, **kwargs):
         r'/value_generator_handler/<generator_id>',
         resources.ValueGeneratorHandler),
     get_redirect_route(r'%s' % feconf.FRACTIONS_LANDING_PAGE_URL,
-                       custom_landing_pages.FractionLandingPage),
-    get_redirect_route(r'%s' % feconf.LIBRARY_INDEX_URL, library.LibraryPage),
+                       custom_landing_pages.FractionLandingPageHandler),
```

Comment:
Break all these multiple lines after `(`. Ditto below.

Reply:
@apb7 I am sorry but how do you want me to break up this single line as it is already broken and it is passing the linting tests.

Reply:
Break after `get_indirect_route(`, that is do something like:
```
get_indirect_route(
    r'%s' % feconf.FRACTIONS_LANDING_PAGE_URL,
    custom_landing_pages.FractionLandingPageHandler)
```
Similarly for others below this.

Reply:
@apb7 is it ok now?